awpy/analytics/map_control.py

```diff
@@ -5,7 +5,7 @@
 
     Example notebook:
 
-    github.com/pnxenopoulos/awpy/blob/main/examples/05_Map_Control_Calculations_And_Visualizations.ipynb
+    _path_to_awpy_repo/examples/05_Map_Control_Calculations_And_Visualizations.ipynb
```

Comment:
I feel we can keep this as the actual link. Should be nice to have the clickable link show up in the documentation. Just silence the line too long warnings here.

Reply:
Yup changed this back to the full length URL and added the ignore line length. Will this link automatically show up as clickable in the documentation or is there something I need to do for that?